src/main/webapp/js/components/user-managment/roles.jsx

```diff
@@ -1,14 +1,37 @@
-import React from 'react';
-
-const Roles = () => (
-  <div className="container-fluid">
-    <h1>
-      Roles Roles Roles Roles
-      <br />
-      Placeholder Placeholder Placeholder Placeholder
-      <br />
-    </h1>
-  </div>
-);
-
-export default Roles;
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { fetchEntity } from '../../actions';
+import EntityTable from '../entities/entity-table';
+import { ROLE_ENTITY, PERMISSION_ENTITY } from '../../utils/entity-types';
+import roleFieldConfig from '../../utils/field-configs';
+
+class Roles extends Component {
+  componentDidMount() {
+    this.props.fetchEntity(ROLE_ENTITY);
```

Comment:
actually role entity will be fetched in EntityTable component, so you can remove this, just fetch the permissions